config/routes.rb

```diff
@@ -10,4 +10,5 @@
   get "/logout", to: "sessions#destroy"
   resources :users
   resources :account_activations, only: :edit
+  resources :password_resets, only: [:new, :create, :edit, :update]

```

Comment:
```suggestion
  resources :password_resets, only: %i(new create edit update)

```
cơ mà em có thể tìm hiểu except và dùng thay vì only á nhé